app/src/test/java/org/wikipedia/test/ParcelableTest.java

```diff
@@ -35,12 +34,4 @@
 
         TestParcelUtil.test(historyEntry);
     }
-
-    @Test public void testPageImage() throws Throwable {
-        WikiSite wiki = WikiSite.forLanguageCode("en");
-        PageTitle title = new PageTitle("Talk", "India", wiki);
-        PageImage pageImage = new PageImage(title, "Testing image");
-
-        TestParcelUtil.test(pageImage);
-    }
```

Comment:
If the previous `PageImage` is `Parcelable`, should we add `@Parcelize` to the `PageImage`?

Reply:
@cooltey I will need to add 'kotlin-android-extensions' plugin to do that... are we ready right now or should we do it when we have a real need for Parcelizing..

Reply:
(it should be the `kotlin-parcelize` plugin, since `kotlin-android-extensions` is deprecated)

Reply:
@dbrant Should I add it now then?